src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -117,47 +117,27 @@ class LicenceConditionService(
       .orElseThrow { EntityNotFoundException("$licenceId") }
 
     val username = SecurityContextHolder.getContext().authentication.name
-    val submittedAdditionalConditions =
+    val submittedConditions =
       request.additionalConditions.transformToEntityAdditional(licenceEntity, request.conditionType)
 
-    val newAdditionalConditions =
-      submittedAdditionalConditions.filter { licenceEntity.additionalConditions.none { submittedCondition -> submittedCondition.conditionCode == it.conditionCode } }
-        .map { newAdditionalCondition ->
-          newAdditionalCondition.copy(
-            expandedConditionText = newAdditionalCondition.conditionText,
-            licence = licenceEntity,
-          )
-        }
+    val existingConditions = licenceEntity.additionalConditions
 
-    val removedAdditionalConditionsList =
-      licenceEntity.additionalConditions.filter {
-        it.conditionType == request.conditionType && submittedAdditionalConditions.none { submittedCondition -> submittedCondition.conditionCode == it.conditionCode }
```

Comment:
not sure why this checks on conditionType whereas the other ones don't. 
I think we have unique codes across AP and PSS so this probably isn't necessary (I've kept it in)